contrib/pkg/deprovision/deprovision.go

```diff
@@ -116,13 +101,3 @@ To run the generic destroyer, use the --metadata-json-secret-name parameter.`,
 	cmd.AddCommand(NewDeprovisionNutanixCommand(logLevel))
 	return cmd
 }
-
-var ConfigureCreds = map[string]func(client.Client, *types.ClusterMetadata){
```

Comment:
moved to the new pkg/creds (this was what allowed me to knock down the circular import)